src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -28,6 +31,19 @@ public class ParserUtilTest {
     private static final String VALID_TAG_2 = "neighbour";
 
     private static final String WHITESPACE = " \t\r\n";
+    private static final String VALID_DEADLINE_WITH_TIME = "25-10-2023 16:00";
+    private static final String VALID_DEADLINE_WITHOUT_TIME = "25-10-2023";
```

Comment:
Just a question: Is it supposed to be intelligent enough to corroborate with real life data, or is it purely formatting?
For example, suppose I put the date for 29th of February on a non leap year, with the correct format, is it supposed to throw exceptions?

Reply:
I believe that the internal LocalDataTime object used will check for the validity of the input datetime, but we could add examples of edge cases that are invalid dates to the test cases to test if the parsing actually works.